content/kubermatic/main/data/ports.md

```diff
@@ -0,0 +1,16 @@
+
+| Source                  | Destination                 | Expose Strategy | Ports             | Purpose                                              |
+|-------------------------|-----------------------------|-----------------|-------------------|------------------------------------------------------|
+| KKP Users               | Master Ingress Controller   | Any             | 443*              | Access to KKP Dashboard                              |
+| KKP Operator            | Seed cluster Kubernetes API | Any             | 6443*             | Operator access                                      |
+| Kubermatic API          | Seed cluster Kubernetes API | Any             | 6443*             | Operator access                                      |
+| Kubermatic API          | Seed cluster nodeport-proxy | Tunneling       | 6443              | Access to User Cluster API Endpoints                 |
+| Kubermatic API          | Seed cluster nodeport-proxy | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints                 |
+| Kubermatic API          | Seed cluster nodeport-proxy | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints                 |
+| Seed controller manager | Cloud Provider API          | Any             | provider specific | Cloud provider api access                            |
+| User cluster nodes      | Seed cluster nodeport-proxy | Tunneling       | 6443, 8088        | Access to User Cluster API Endpoints and Konnecitivy |
+| User cluster nodes      | Seed cluster nodeport-proxy | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints and Konnecitivy |
+| User cluster nodes      | Seed cluster nodeport-proxy | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints and Konnecitivy |
+| KKP Users               | Seed cluster nodeport-proxy | Tunneling       | 6443              | Access to User Cluster API Endpoints                 |
+| KKP Users               | Seed cluster nodeport-proxy | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints                 |
+| KKP Users               | Seed cluster nodeport-proxy | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints                 |
```

Comment:
| Source                  | Destination                 | Expose Strategy | Ports             | Purpose                                              |
|-------------------------|-----------------------------|-----------------|-------------------|------------------------------------------------------|
| Master Cluster Nodes           | Seed Cluster Kubernetes API | Any             | 6443*             | Provisioning and managing KKP Seeds                                      
| Master Cluster Nodes        | Seed Cluster Load Balancers | Tunneling       | 6443              | Access to User Cluster API Endpoints                 |
| Master Cluster Nodes          | Seed Cluster Nodes or Load Balancers | NodePort        | 30000-32767**     | Access to User Cluster Kubernetes API Endpoints                 |
| Master Cluster Nodes          | Seed Cluster Load Balancers | LoadBalancer    | 30000-32767**     | Access to User Cluster Kubernetes API Endpoints                 |
| Seed Cluster Nodes | Cloud Provider APIs          | Any             | provider specific | Cloud provider api access to provision cloud resources (e.g. virtual machines)                            |
| User Cluster Nodes      | Seed Cluster Load Balancers | Tunneling       | 6443, 8088        | Access to User Cluster API Endpoints and Konnectivity |
| User Cluster Nodes      | Seed Cluster Nodes or Load Balancers  | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints and Konnectivity |
| User Cluster Nodes      | Seed Cluster Load Balancers | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints and Konnectivity |
| KKP Users               | Master Load Balancers  | Any             | 443*              | Access to KKP Dashboard                              |
| KKP Users               | Seed Cluster Load Balancers | Tunneling       | 6443              | Access to User Cluster API Endpoints                 |
| KKP Users               | Seed Cluster Nodes or Load Balancers | NodePort        | 30000-32767**     | Access to User Cluster API Endpoints                 |
| KKP Users               | Seed Cluster Load Balancers | LoadBalancer    | 30000-32767**     | Access to User Cluster API Endpoints                 |